Minor fix: Remove Skeleton's Freeagent mindrole

JustAdler opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Removes Skeleton's Freeagent mindrole 
## Why we need to add this
![halfway-there-bon-jovi](https://github.com/user-attachments/assets/bce202e7-45ba-4d3c-9df9-7bb525bb4208)
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: Aedler
- fix: Removed skeleton's mob freeagent mindrole

JustAdler committed:
skele

Happyrobot33 commented:
Uhhh

None of the pr description is filled out. Silly

JustAdler commented:
> Uhhh
> 
> None of the pr description is filled out. Silly

eeeeeeeh

CringeCursed commented:
Why changelog validator dont yells about pr body(

KillerTheFlareon commented:
> > Uhhh
> > None of the pr description is filled out. Silly
> 
> eeeeeeeh

lazy bastard, fill it out!

JustAdler commented:
> Why changelog validator dont yells about pr body(

well it done goofed now

CringeCursed commented:
> > Why changelog validator dont yells about pr body(
> 
> well it done goofed now

💀

CringeCursed commented:
> > Why changelog validator dont yells about pr body(
> 
> well it done goofed now



> > Why changelog validator dont yells about pr body(
> 
> well it done goofed now



> > Why changelog validator dont yells about pr body(
> 
> well it done goofed now

idk it works fine with others PR.

can you send me ur pr body(Not formated obviously)  containment?

JustAdler commented:
> > > Why changelog validator dont yells about pr body(
> > 
> > 
> > well it done goofed now
> 
> > > Why changelog validator dont yells about pr body(
> > 
> > 
> > well it done goofed now
> 
> > > Why changelog validator dont yells about pr body(
> > 
> > 
> > well it done goofed now
> 
> idk it works fine with others PR.
> 
> can you send me ur pr body(Not formated obviously) containment?

```
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Removes Skeleton's Freeagent mindrole 
## Why we need to add this
![halfway-there-bon-jovi](https://github.com/user-attachments/assets/bce202e7-45ba-4d3c-9df9-7bb525bb4208)
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.-->

:cl: Aedler
- fix: Removed skeleton's mob freeagent mindrole

```

CringeCursed commented:
> <!--
> If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.-->


If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

remove that with <!--

JustAdler committed:
Merge branch 'Starlight' into skele

JustAdler commented:
huh, github prompts me to update the branch constantly, bringing in another validator check fail

JustAdler committed:
Merge branch 'Starlight' into skele

CringeCursed commented:
> huh, github prompts me to update the branch constantly, bringing in another validator check fail

shush!
